cluster-scope/base/rbac.authorization.k8s.io/clusterroles/project-monitoring-edit/clusterrole.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: project-monitoring-edit
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+rules:
+  - verbs:
+      - '*'
+    apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - servicemonitors
+      - podmonitors
+      - prometheusrules
+      - alertmanagerconfigs
```

Comment:
This is based on the existing [ClusterRoles defined in the Red Hat documentation](https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects). 

```bash
$ oc get clusterrole/monitoring-edit -o yaml | grep rules: -A 10
rules:
- apiGroups:
  - monitoring.coreos.com
  resources:
  - servicemonitors
  - podmonitors
  - prometheusrules
  verbs:
  - '*'

$ oc get clusterrole/alert-routing-edit -o yaml | grep rules: -A 10
rules:
- apiGroups:
  - monitoring.coreos.com
  resources:
  - alertmanagerconfigs
  verbs:
  - '*'
```